test/recipes/80-test_cmp_http.t

```diff
@@ -293,7 +293,7 @@ sub start_server {
                           $args ? $args : ()]), display => 1);
     print "Current directory is ".getcwd()."\n";
     print "Launching $server_name server: $cmd\n";
-    my $pid = open($server_fh, "$cmd 2>error.txt |");
+    my $pid = open($server_fh, "$cmd 2>".result_dir()."/error.txt |");
```

Comment:
Will the use of `/` here work on Windows? i.e. does Perl's `open()` use `cmd.exe` (in which case it won't) or does it call the Windows API directly (in which case it will)

Reply:
Hmm, testing suggests that it should (at least in an N=1 case)

Reply:
yes, I think windows are fine with both variants '/' vs. '\' at least on command line. also test results confirm that: https://github.com/openssl/openssl/actions/runs/12694643424/job/35384862742?pr=26363